docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -124,6 +124,99 @@ elastic-agent-fj2z9             1/1     Running   0          81m
+
+. In the manifest file, modify the `agent-node-datastreams` ConfigMap and adjust inputs:
++
+--
+* `kubernetes-cluster-metrics` input:
+** If used https to access kube-state-metrics, to all `kubernetes.state_*` dataset should be added:
++
+[source,yaml]
+------------------------------------------------
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  ssl.certificate_authorities:
+    - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+------------------------------------------------
+* `kubernetes-node-metrics` input:
+** `kubernetes.controllermanager` datastream condition should be change to:
++
+[source,yaml]
+------------------------------------------------
+condition: ${kubernetes.labels.app} == 'kube-controller-manager'
+------------------------------------------------
+** `kubernetes.scheduler` datastream condition should be changed to:
++
+[source,yaml]
+------------------------------------------------
+condition: ${kubernetes.labels.app} == 'openshift-kube-scheduler'
+------------------------------------------------
+** `kubernetes.proxy` datastream configutation should look like:
++
+[source,yaml]
+------------------------------------------------
+- data_stream:
+    dataset: kubernetes.proxy
+    type: metrics
+  metricsets:
+    - proxy
+  hosts:
+    - 'localhost:29101'
+  period: 10s
+------------------------------------------------
+** to all datastreams that connnect to `https://${env.NODE_NAME}:10250` should be added:
++
+[source,yaml]
+------------------------------------------------
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  ssl.certificate_authorities:
+    - /path/to/ca-bundle.crt
+------------------------------------------------
+NOTE: `ca-bundle.crt` can be any CA bundle that contains the issuer of the certificate used in the Kubelet API.
+According to each specific installation of Openshift this can be found either in `secrets` or in `configmaps`.
+In some installations it can be available as part of the service account secret, in
+`/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`.
+In case of using https://github.com/openshift/installer/blob/master/docs/user/gcp/install.md[Openshift installer]
+for GCP then the following `configmap` can be mounted in Metricbeat Pod and use `ca-bundle.crt`
+in `ssl.certificate_authorities`:
++
+[source,shell]
+-----
+Name:         kubelet-serving-ca
+Namespace:    openshift-kube-apiserver
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+ca-bundle.crt:
+-----
+--
+. Grant the `elastic-agent` service account access to the privileged SCC:
++
+[source,shell]
+-----
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:elastic-agent
+-----
++
+This command enables the container to be privileged as an administrator for
+OpenShift.
+
+. If namespace, where elastic-agent is running, has `"openshift.io/node-selector"` annotation set, override node
```

Comment:
Isn't this change too generic? This might affect other workloads too in the cluster no? Any alternatives here?

Reply:
I took this part from the metricbeat documentation - https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html#_red_hat_openshift_configuration

actually this can be just dropped, because on Openshift v4 in default namespace for elastic-agent - `kube-system` this annotation is not set (openshift on gcp setup), so elastic-agent was deployed on both worker and master nodes without any changes. I kept this just as a possible explanation why elastic-agent might be not deployed on all nodes (if running not in standard namespace or there are some differences on other environments)

wdyt about:
```suggestion
. If namespace, where elastic-agent is running, has `"openshift.io/node-selector"` annotation set, elastic-agent might get running not on all nodes. In this case consider overriding node selector for the namespace to allow scheduling on any node:
```

Reply:
Thanks for elaborating, didn't remember we had this in Beats too. I think your suggestion is safer, so good to go!

Reply:
done https://github.com/elastic/observability-docs/pull/1498/commits/1a048d1f1acda4f6730c9b509e3ec5a3c4705b2a